src/toil/jobStores/abstractJobStore.py

```diff
@@ -223,6 +223,9 @@ def exists(self, jobStoreID):
     # One year should be sufficient to finish any pipeline ;-)
     publicUrlExpiration = timedelta(days=365)
 
+    def getEnv(self):
+        return ['AZURE_ACCOUNT_KEY'] #not sure what other env variables to ask for
```

Comment:
This is an Azure-specific detail that has no place in the abstract job store. Override this method in the Azure job store and change this implementation to return an empty collection.